.env.example

```diff
@@ -1,14 +1,14 @@
 # App
 HTTP_PORT=8080
-DB_PORT=8123
+DB_PORT=8443
```

Comment:
`8123` is the standard HTTP port for a [newly baked local Clickhouse instance](https://clickhouse.com/docs/en/interfaces/http#) (if you don't change server config), and `8443` is for HTTPS. So I propose to leave the current value

Reply:
Wouldn't HTTPS be preferred by default over HTTP if possible? Perhaps I'm missing something

Reply:
it makes no sense for a local instance that runs as a `docker-compose` service (the standard and fastest approach for this project). If you have an external Clickhouse (on another host or in a different environment), of course, HTTPS would be the best option for you

Reply:
Oh! I completely missed that part. I actually went ahead and create a ClickHouse instance, and didn't notice we are actually running one locally as part of the compose project. I agree.